### PR TITLE
*: remove TiDB Binlog after deprecation

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Starting from TiDB v8.4.0, TiDB Binlog is removed after deprecation. This PR removes and updates documents related to removing TiDB Binlog.

**Note:** 
- The TiDB Binlog line in [TiDB 功能概览 | PingCAP 文档中心](https://docs.pingcap.com/zh/tidb/dev/basic-features) will be updated in a separate PR that adds a new v8.4 column to `basic-features.md` and updates the status of all affected features. This PR doesn't update it.
- I've added `aliases` for 21 deleted `.md` files in PR https://github.com/pingcap/docs-cn/pull/18625, to avoid broken 404 links. All the URLs of deleted files will be redirected to those links of v8.3. 
- The original PR https://github.com/pingcap/docs-cn/pull/18558 involves too many conflicts because of many commits behind the upstream/master. So I closed it and create this PR. In addition to the updates by #18558, I made more updates in this PR.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

File: releases/release-8.3.0.md [hunk 1]
```diff
@@ -233,13 +233,13 @@ TiDB 版本：8.3.0
 
 ### 系统表
 
-* 在系统表 [`INFORMATION_SCHEMA.PROCESSLIST`](/information-schema/information-schema-processlist.md) 和 [`INFORMATION_SCHEMA.CLUSTER_PROCESSLIST`](/information-schema/information-schema-processlist.md#cluster_processlist) 中新增 `ROWS_AFFECTED` 字段，用于显示 DML 语句当前影响的数据行数。[#46889](https://github.com/pingcap/tidb/issues/46889) @[lcwangchao](https://github.com/lcwangchao) 
+* 在系统表 [`INFORMATION_SCHEMA.PROCESSLIST`](/information-schema/information-schema-processlist.md) 和 [`INFORMATION_SCHEMA.CLUSTER_PROCESSLIST`](/information-schema/information-schema-processlist.md#cluster_processlist) 中新增 `ROWS_AFFECTED` 字段，用于显示 DML 语句当前影响的数据行数。[#46889](https://github.com/pingcap/tidb/issues/46889) @[lcwangchao](https://github.com/lcwangchao)
```
Comment (Benjamin2037) [inline releases/release-8.3.0.md:236]: 这个好像和 Binlog 文档没有关联！

Comment (D3Hunter) [reply releases/release-8.3.0.md:236]: 这里只是去掉了个空格吧

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: releases/release-8.3.0.md (hunk 1 above)
Comment (Benjamin2037) [reply releases/release-8.3.0.md:236]: yes

**Comment by Benjamin2037** [comment]
https://docs.pingcap.com/zh/tidb/stable/sql-statement-show-pump-status#%E7%A4%BA%E4%BE%8B
https://docs.pingcap.com/zh/tidb/stable/sql-statement-show-drainer-status
https://docs.pingcap.com/zh/tidb/stable/sql-statement-change-pump
https://docs.pingcap.com/zh/tidb/stable/sql-statement-change-drainer
@lilin90 这几个相关语句文档也需要删除掉

**Comment by Benjamin2037** [comment]
https://docs.pingcap.com/zh/tidb/stable/sql-statement-change-drainer
696503f4-40a2-4e89-bda1-3a2d581c91a3
这几个语句的说明可能也需要删除

**Comment by lilin90** [comment]
> https://docs.pingcap.com/zh/tidb/stable/sql-statement-change-drainer 696503f4-40a2-4e89-bda1-3a2d581c91a3 这几个语句的说明可能也需要删除

Done in https://github.com/pingcap/docs-cn/pull/18576/commits/f5d3bd030230f58d70c4e71714c1efa6c2ac2a2b.

File: alert-rules.md [hunk 2]
```diff
@@ -121,12 +121,11 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
     TiDB 服务中发生的事件数量。当出现以下事件的时候会报警：
 
     1. start：TiDB 服务启动。
-    2. hang：当发生了 Critical 级别的事件时（目前只有 Binlog 写不进去一种情况），TiDB 进入 `hang` 模式，并等待人工 Kill。
+    2. hang：当发生了 Critical 级别的事件时，TiDB 进入 `hang` 模式，并等待人工 Kill。
 
 * 处理方法：
 
     * 重启 TiDB 以恢复服务。
```
Comment (lilin90) [inline alert-rules.md:128]:
```suggestion
    重启 TiDB 以恢复服务。
```

File: grafana-tidb-dashboard.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
 - Panic And Critical Error：TiDB 中出现的 Panic、Critical Error 数量。
 - Time Jump Back OPS：每个 TiDB 实例上每秒操作系统时间回跳的次数。
 - Get Token Duration：每个连接获取 Token 的耗时。
-- Skip Binlog Count：TiDB 写入 Binlog 失败的数量。
+- Skip Binlog Count：TiDB 写入 Binlog 失败的数量。从 v8.4.0 开始，TiDB Binlog 已移除，该指标不再有计数。
```
[the remark this answers is not in the document]
Comment (lilin90) [reply grafana-tidb-dashboard.md:54]: 已确认保留

File: identify-slow-queries.md [hunk 4]
```diff
@@ -101,7 +101,7 @@ Slow Query 基础信息：
 * `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
 * `Write_size`：表示事务提交时写 key 或 value 的总大小。
 * `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。每个 Region 会触发一次远程过程调用。
-* `Wait_prewrite_binlog_time`：表示事务提交时用于写 binlog 的时间。
+* `Wait_prewrite_binlog_time`：表示事务提交时用于写 binlog 的时间。从 v8.4.0 开始，TiDB Binlog 已移除，不再有相关时间。
```
[the remark this answers is not in the document]
Comment (lilin90) [reply identify-slow-queries.md:104]: 已确认保留

File: system-variables.md [hunk 5]
```diff
@@ -601,7 +601,7 @@ mysql> SELECT * FROM t1;
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：布尔型
 - 默认值：`OFF`
-- 该变量表示是否使用 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)（已废弃）。
+- 该变量表示是否使用 [TiDB Binlog](https://docs.pingcap.com/zh/tidb/v8.3/tidb-binlog-overview)。从 v8.4 开始，TiDB Binlog 已移除，该变量无效。
```
[the remark this answers is not in the document]
Comment (lilin90) [reply system-variables.md:604]: 已确认删除

File: releases/release-8.3.0.md [hunk 6]
```diff
@@ -233,13 +233,13 @@ TiDB 版本：8.3.0
...
 
 ## 废弃功能
 
 * 以下为从 v8.3.0 开始已废弃的功能：
 
-    * 从 v7.5.0 开始，[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 的数据同步功能被废弃。从 v8.3.0 开始，TiDB Binlog 被完全废弃，并计划在未来版本中移除。如需进行增量数据同步，请使用 [TiCDC](/ticdc/ticdc-overview.md)。如需按时间点恢复 (point-in-time recovery, PITR)，请使用 [PITR](/br/br-pitr-guide.md)。
+    * 从 v7.5.0 开始，[TiDB Binlog](https://docs.pingcap.com/zh/tidb/v8.3/tidb-binlog-overview) 的数据同步功能被废弃。从 v8.3.0 开始，TiDB Binlog 被完全废弃，并计划在未来版本中移除。如需进行增量数据同步，请使用 [TiCDC](/ticdc/ticdc-overview.md)。如需按时间点恢复 (point-in-time recovery, PITR)，请使用 [PITR](/br/br-pitr-guide.md)。
```
Comment (hfxsd) [inline releases/release-8.3.0.md:242]: 这里的“未来版本“要改成 v8.4.0 吗

File: tidb-configuration-file.md [hunk 7]
```diff
@@ -504,7 +504,7 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + TiDB 单个事务大小限制
 + 默认值：104857600
 + 单位：Byte
-+ 单个事务中，所有 key-value 记录的总大小不能超过该限制。该配置项的最大值不超过 `1099511627776`（表示 1TB）。注意，如果使用了以 `Kafka` 为下游消费者的 `binlog`，如：`arbiter` 集群，该配置项的值不能超过 `1073741824`（表示 1GB），因为这是 `Kafka` 的处理单条消息的最大限制，超过该限制 `Kafka` 将会报错。
++ 单个事务中，所有 key-value 记录的总大小不能超过该限制。该配置项的最大值不超过 `1099511627776`（表示 1TB）。注意，如果使用了以 `Kafka` 为下游消费者的 `binlog`，该配置项的值不能超过 `1073741824`（表示 1GB），因为这是 `Kafka` 的处理单条消息的最大限制，超过该限制 `Kafka` 将会报错。
```
Comment (hfxsd) [inline tidb-configuration-file.md:507]: 有 `binlog`，是否也要删掉
```suggestion
+ 单个事务中，所有 key-value 记录的总大小不能超过该限制。该配置项的最大值不超过 `1099511627776`（表示 1TB）。
```

File: maintain-tidb-using-tiup.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ tiup cluster list
 
 ## 启动集群
 
-启动集群操作会按 PD -> TiKV -> Pump -> TiDB -> TiFlash -> Drainer -> TiCDC -> Prometheus -> Grafana -> Alertmanager 的顺序启动整个 TiDB 集群所有组件：
+启动集群操作会按 PD -> TiKV -> TiDB -> TiFlash -> Drainer -> TiCDC -> Prometheus -> Grafana -> Alertmanager 的顺序启动整个 TiDB 集群所有组件：
```
Comment (hfxsd) [inline maintain-tidb-using-tiup.md:23]:
```suggestion
启动集群操作会按 PD -> TiKV -> TiDB -> TiFlash -> TiCDC -> Prometheus -> Grafana -> Alertmanager 的顺序启动整个 TiDB 集群所有组件：
```

File: maintain-tidb-using-tiup.md [hunk 9]
```diff
@@ -187,7 +187,7 @@ tiup cluster rename ${cluster-name} ${new-name}
 
 ## 关闭集群
 
-关闭集群操作会按 Alertmanager -> Grafana -> Prometheus -> TiCDC -> Drainer -> TiFlash -> TiDB -> Pump -> TiKV -> PD 的顺序关闭整个 TiDB 集群所有组件（同时也会关闭监控组件）：
+关闭集群操作会按 Alertmanager -> Grafana -> Prometheus -> TiCDC -> Drainer -> TiFlash -> TiDB -> TiKV -> PD 的顺序关闭整个 TiDB 集群所有组件（同时也会关闭监控组件）：
```
Comment (hfxsd) [inline maintain-tidb-using-tiup.md:190]:
```suggestion
关闭集群操作会按 Alertmanager -> Grafana -> Prometheus -> TiCDC -> TiFlash -> TiDB -> TiKV -> PD 的顺序关闭整个 TiDB 集群所有组件（同时也会关闭监控组件）：
```

File: releases/release-8.3.0.md (hunk 6 above)
Comment (lilin90) [reply releases/release-8.3.0.md:242]: 这是一个通用说法，其它废弃 feature 也会这么表述，可以保持现状。

File: tidb-configuration-file.md (hunk 7 above)
Comment (lilin90) [reply tidb-configuration-file.md:507]: 不是指 TiDB Binlog，应该就不需要删除，保持现状即可。

Comment (D3Hunter) [inline tidb-configuration-file.md:507]:
```suggestion
+ 单个事务中，所有 key-value 记录的总大小不能超过该限制。该配置项的最大值不超过 `1099511627776`（表示 1TB）。
```

Comment (lilin90) [reply tidb-configuration-file.md]: Confirmed with @D3Hunter, deleted via https://github.com/pingcap/docs-cn/pull/18576/commits/bba75f618a7874935861ced69d28f7bb96934d0a.

**Comment by lilin90** [comment]
@Benjamin2037 Would you please take a look at the two new commits that delete more descriptions about Pump and Drainer? https://github.com/pingcap/docs-cn/pull/18576/commits/ca472d75a0927317821cc55e8552c29aa78fd022 and https://github.com/pingcap/docs-cn/pull/18576/commits/3b8bf6f2b72fa8ae28c7ffec374a5e52cd8d2834

**Pull request merged**